stock_orderpoint_procure_location/models/stock_warehouse_orderpoint.py

```diff
@@ -13,11 +13,34 @@ class Orderpoint(models.Model):
         domain="[('usage', '=', 'internal')]",
     )
 
-    def _prepare_procurement_values(self, product_qty, date=False, group=False):
+    def _prepare_procurement_values(self, date=False, group=False):
         """Set the procure location"""
-        res = super(Orderpoint, self)._prepare_procurement_values(
-            product_qty, date, group
-        )
+        res = super(Orderpoint, self)._prepare_procurement_values(date, group)
         if self.procure_location_id:
             res["procure_location_id"] = self.procure_location_id
         return res
+
+    def _quantity_in_progress(self):
+        res = super()._quantity_in_progress()
+        not_count_states = ["done", "cancel"]
+        for orderpoint in self:
+            if orderpoint.id == 780:
+                pass
+            for sm in (
+                self.env["purchase.order.line"]
+                .search(
+                    [
+                        ("state", "=", "purchase"),
+                        ("product_id", "=", orderpoint.product_id.id),
+                    ]
+                )
+                .mapped("move_ids")
+                .filtered(
+                    lambda m: m.location_dest_id == orderpoint.procure_location_id
+                    and m.state not in not_count_states
+                )
+            ):
+                res[orderpoint.id] += sm.product_uom._compute_quantity(
```

Comment:
Could you check why this is not covered by tests ?

Reply:
I will check that